### PR TITLE
Add Warning Around Import Command

### PR DESCRIPTION
**Pull request by jconstance-amplify**

### CODE REVIEW

File: bin/tf [hunk 1]
```diff
@@ -87,6 +87,16 @@ def exec_tf_command(
     if command == "init":
         shutil.rmtree(os.path.join(path, ".terraform"), ignore_errors=True)
 
+    if command == "import":
```
Comment (kelibezhani) [inline bin/tf]: Do you have a check verifying that it is actually en existing state file, or will we get this message for every import command?

Comment (jconstance-amplify) [reply bin/tf]: For every import command. I'm not sure if it's worth the effort to make that kind of check. You'd have to check if it's merged, essentially.

Comment (kelibezhani) [reply bin/tf]: I believe if path exists in pipeline is a good indicator that state file is not new.

Comment (jconstance-amplify) [reply bin/tf]: But we're removing pipelines.

Comment (kelibezhani) [reply bin/tf]: There is a scenario of running some sort of script to import multiple resources in state file.

Comment (jconstance-amplify) [reply bin/tf]: You could use `yes`

Comment (kelibezhani) [reply bin/tf]: 👍

**Pull request merged**